src/main/webapp/css/srp_style.css

```diff
@@ -42,28 +42,64 @@ h1 {
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  max-height: 260px;
+  overflow: scroll;
+  font: 16px Roboto, Arial, sans-serif;
+  margin-top: 20px;
 }
 
-.card .card-title {
-  font: 300 20px Roboto, Arial, sans-serif;
+.card::-webkit-scrollbar {
+  width: 8px;
```

Comment:
nit: Since the scrollbar is 8px, I think it looks better if instead of .card padding being 24px all around, it's slightly adjusted for the right hand side with `24px 16px 24px 24px` but this is slightly eyeballing and maybe not completely precise